app/scripts/controllers/preferences-controller.ts

```diff
@@ -114,176 +123,356 @@ export type Preferences = {
+    useMultiAccountBalanceChecker: true,
+    useSafeChainsListValidation: true,
+    // set to true means the dynamic list from the API is being used
+    // set to false will be using the static list from contract-metadata
+    useTokenDetection: true,
+    useNftDetection: true,
+    use4ByteResolution: true,
+    useCurrencyRateCheck: true,
+    useRequestQueue: true,
+    openSeaEnabled: true,
+    securityAlertsEnabled: true,
+    watchEthereumAccountEnabled: false,
+    bitcoinSupportEnabled: false,
+    bitcoinTestnetSupportEnabled: false,
+    ///: BEGIN:ONLY_INCLUDE_IF(keyring-snaps)
+    addSnapAccountEnabled: false,
+    ///: END:ONLY_INCLUDE_IF
+    advancedGasFee: {},
+    featureFlags: {},
+    incomingTransactionsPreferences: {
+      ...mainNetworks,
+      ...testNetworks,
+    },
+    knownMethodData: {},
+    currentLocale: '',
+    identities: {},
+    lostIdentities: {},
+    forgottenPassword: false,
+    preferences: {
+      autoLockTimeLimit: undefined,
+      showExtensionInFullSizeView: false,
+      showFiatInTestnets: false,
+      showTestNetworks: false,
+      smartTransactionsOptInStatus: null, // null means we will show the Smart Transactions opt-in modal to a user if they are eligible
+      showNativeTokenAsMainBalance: false,
+      useNativeCurrencyAsPrimaryCurrency: true,
+      hideZeroBalanceTokens: false,
+      petnamesEnabled: true,
+      redesignedConfirmationsEnabled: true,
+      redesignedTransactionsEnabled: true,
+      featureNotificationsEnabled: false,
+      isRedesignedConfirmationsDeveloperEnabled: false,
+      showConfirmationAdvancedDetails: false,
+      showMultiRpcModal: false,
+      shouldShowAggregatedBalancePopover: true, // by default user should see popover;
+      tokenSortConfig: {
+        key: 'tokenFiatAmount',
+        order: 'dsc',
+        sortCallback: 'stringNumeric',
+      },
+    },
+    // ENS decentralized website resolution
+    ipfsGateway: IPFS_DEFAULT_GATEWAY_URL,
+    isIpfsGatewayEnabled: true,
+    useAddressBarEnsResolution: true,
+    // Ledger transport type is deprecated. We currently only support webhid
+    // on chrome, and u2f on firefox.
+    ledgerTransportType: window.navigator.hid
+      ? LedgerTransportTypes.webhid
+      : LedgerTransportTypes.u2f,
+    snapRegistryList: {},
+    theme: ThemeType.os,
+    ///: BEGIN:ONLY_INCLUDE_IF(keyring-snaps)
+    snapsAddSnapAccountModalDismissed: false,
+    ///: END:ONLY_INCLUDE_IF
+    useExternalNameSources: true,
+    useTransactionSimulations: true,
+    enableMV3TimestampSave: true,
+    // Turning OFF basic functionality toggle means turning OFF this useExternalServices flag.
+    // Whenever useExternalServices is false, certain features will be disabled.
+    // The flag is true by Default, meaning the toggle is ON by default.
+    useExternalServices: true,
+    // from core PreferencesController
+    isMultiAccountBalancesEnabled: true,
+    showIncomingTransactions: {
```

Comment:
This seems to overlap with 
```
    incomingTransactionsPreferences: {
      ...mainNetworks,
      ...testNetworks,
    },
```
Can we combine both?

Reply:
Good to know it is taken care of. Do ping me if you need some 👁️ for that ticket to refine or review ~